Color_detection/Readme.md

```diff
@@ -0,0 +1,20 @@
+
+# Colour Detection using Pandas and OpenCV
+
+In this color detection Python project, we are going to build an application through which you can automatically get the name of the color by clicking on them. So for this, we will have a data file that contains the color name and its values. Then we will calculate the distance from each color and find the shortest one.
+
+# Steps Used
+##-Importing required package and load the image.
```

Comment:
This is how it's getting rendered:
![image](https://user-images.githubusercontent.com/43691873/111168782-676bbf00-85c8-11eb-99c1-fdcb8fa2cc42.png)

Try this hierarchy:
```
# Steps Used
- Importing ...
- Reading ...
- Set a mouse ...
    - First
    - Second
 - Draw 
 - Calculate
 - Display
```

Reply:
Done @kaustubhgupta